gradle/libs.versions.toml

```
@@ -18,7 +18,7 @@ kotlin = "1.6.21"
 kotlinCoroutines = "1.5.0"
 mockito = "4.6.1"
 mockitoKotlin = "4.0.0"
-telemetryGenerator = "1.0.58"
+telemetryGenerator = "1.0.59"
```

```suggestion
telemetryGenerator = "1.0.61"
```

please pull in latest